MdePkg/Include/Pi/PiHob.h

```diff
@@ -332,6 +333,25 @@ typedef struct {
   UINT64                         ResourceLength;
 } EFI_HOB_RESOURCE_DESCRIPTOR;
 
+/// PI Spec Status: Pending.
+/// This change is checked in as a code first approach. The PI spec will be updated
+/// to reflect this change in the future.
+///
+/// Describes the resource properties of all fixed,
+/// nonrelocatable resource ranges found on the processor
+/// host bus during the HOB producer phase.

```

Comment:
This is the same comment as v1, probably should add a sentence on what distinguishes v2.

Reply:
Updated.